webpack/assets/javascripts/react_app/redux/actionTypeGenerator.js

```diff
@@ -0,0 +1,9 @@
+const actions = ['REQUEST', 'SUCCESS', 'FAILURE'];
+
+const actionTypeGenerator = (subtype, actionTypes = {}) => {
```

Comment:
Can you use esdocs comment style to document this function?
I see it as a key part to understanding how API communication works.

Reply:
Added, thanks